www/common/common-ui-elements.js

```diff
@@ -2510,13 +2510,11 @@ define([
                 } else {
                     next();
                 }
-                return;
-            }
-            if (e.which === 13) {
+              }
+            else if (e.which === 13) {
                 if ($container.find('.cp-icons-element-selected').length === 1) {
                     $container.find('.cp-icons-element-selected').click();
                 }
-                return;
             }
```

Comment:
Try to avoid this type of changes that don't have any impact on the results but only on the code style. I recommend to always run a `git diff` before creating a commit in order to clean unnecessary changes :)